README.md

```diff
@@ -54,61 +79,104 @@ CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s
+```bash
+Loaded policy 'root'
+{
+  "created_roles": {
+    "cucumber:host:<POLICY_ID>/<NAME_OF_HOST>": {
+      "id": "cucumber:host:<POLICY_ID>/<NAME_OF_HOST>",
+      "api_key": "3scdjaz3aq90vn26avdv11s2mf2h2wnwfzw3cbgbja1jspzyc1rdzwqx"
+    }
+  },
+  "version": 1
+}
+```
+
+5. Set the ID and API key as the environment variables CONJUR_AUTHN_LOGIN and CONJUR_AUTHN_API_KEY respectively. 
+    
+    _NOTE:_ In Conjur, it is the standard that hosts are referred to in terms of their full hostname. Therefore, when setting value for the host's environment 
+    variable, the host ID should be prefixed with `host/`. For example, the `host/<NAME_OF_HOST>`, not just `<NAME_OF_HOST>`.
+
+6. Give the variable defined in the previous policy a value like so: `conjur variable values add <POLICY_ID>/<NAME_OF_VARIABLE>`
 
-By default, the Conjur appliance generates and uses self-signed SSL certificates. You'll need to configure
-Java to trust them. You can accomplish this by loading the Conjur certificate into the Java keystore.
+Now that we have identified our host in Conjur, we now need the Java app to trust Conjur. To do this, the Conjur self-signed SSL certificate needs to be added 
+to the app's keystore.  
+
+## Setup Trust Between App and Conjur
+By default, the Conjur appliance generates and uses self-signed SSL certificates (Java-specific certificates known as cacerts). 
+You'll need to configure your Java app to trust them. You can accomplish this by loading the Conjur certificate into the app's keystore.
 First, you'll need a copy of this certificate, which you can get using the [Conjur CLI](https://developer.conjur.net/cli).
-Once you've installed the command line tools, you can run
+Run the following steps from the Conjur CLI to initialize Conjur in order to get Conjur certificates:
 
 ```bash
 conjur init
 ```
 
-and enter the required information at the prompts.  This will save the certificate to a file like `"conjur-mycompany.pem"`
-in your HOME directory.  Java doesn't deal with the *pem* format, so next you'll need to convert it to the *der* format:
+and enter the required information at the prompts (URL of Conjur service and account name).  This will save the certificate to a file like `"conjur-mycompany.pem"`
+in your HOME directory.  Java doesn't work with the *pem* encoding format, so next you'll need to convert it to the *der* format:
+
 
 ```bash
-openssl x509 -outform der -in conjur-yourcompany.pem -out conjur-yourcompany.der
+openssl x509 -outform der -in <PEM_PATH_IN_CLI/conjur-yourcompany.pem> -out conjur-yourcompany.der
 ```
 
-Next, you'll need to locate your JRE home.   On my machine it's `/usr/lib/jvm/java-7-openjdk-amd64/jre/`.  We'll export
+Next, you'll need to locate the JRE home from the process environment running the Java app. On my machine it's `/usr/lib/jvm/java-8-openjdk-amd64/jre'`.  We'll export
```

Comment:
`On my machine` -> `In the case of Java 8 on most standard Linux distributions it's`